libgraph/include/katana/PropertyGraph.h

```diff
@@ -861,6 +877,12 @@ class KATANA_EXPORT PropertyGraph {
   /// Return the number of local edges
   uint64_t NumEdges() const { return topology().NumEdges(); }
 
+  /// Return the number of nodes of the original property graph.
+  virtual uint64_t NumOriginalNodes() const { return NumNodes(); }
+
+  /// Return the number of edges of the original property graph.
+  virtual uint64_t NumOriginalEdges() const { return NumEdges(); }
+
```

Comment:
would be good to group all virtual methods together so it is easier not to forget one when writing a derived class